specification/subscription/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-subscriptions"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-subscriptions"
+  output-folder: "$(typescript-sdks-folder)/sdk/subscriptions/arm-subscriptions"
```

Comment:
should this be "subscription" without an "s".

Reply:
yes I checked the .NET repo, it doesn't have the plural form.